### DISCUSSION TITLE
Change font for Preview

### DISCUSSION THREAD

**Discussion by Decadenona** [jendrikseipp/rednotebook#554] [Q&A]
I don't know if I am in the right place here - but here goes anyway . . .

How does one change the font seen in Preview in Red Note Book please ?
My installation on Linux Mint 20 (Cinnamon) seems resolutely stuck on the installed font and no matter what I do in "Preferences" I do not seem to be able to change it.

**Comment by jendrikseipp** [comment]
Under "Preferences" -> "Preview font" you can select a font. Try for example "Times New Roman" or "Arial" to make sure the setting has an effect.
